layouts/404.html

```diff
@@ -1,10 +1,35 @@
 {{ define "main" }}
-<div class="row justify-content-center">
-  <div class="col-md-12 col-lg-10 col-xl-8">
-    <article>
-      <h1 class="text-center">Page not found :(</h1>
-      <p class="text-center">The page you are looking for doesn't exist or has been moved.</p>
-    </article>
+<section class="section section-sm">
+  <div class="container">
+    <div class="row justify-content-center">
+      <div class="col-md-12 col-lg-8 col-xl-6">
+        <article>
+          <h1 class="text-left">We couldn't find the page you were looking for.</h1>
```

Comment:
```suggestion
          <h1 class="text-left">We couldn't find the page you were looking for. It may have moved, or you might have an incorrect link.</h1>
```